frontend/apps/filemanager/filemanager.lua

```diff
@@ -293,9 +294,18 @@ function FileManager:init()
     table.insert(self, self.menu)
     table.insert(self, FileManagerHistory:new{
         ui = self,
-        menu = self.menu
     })
 
+    self.loaded_modules = {}
+    -- koreader plugins
+    for _,plugin_module in ipairs(PluginLoader:loadPlugins()) do
+        DEBUG("Loaded plugin", plugin_module.name, "at", plugin_module.path)
+        if plugin_module.docless then
```

Comment:
I think it's better to name it as `is_doc_only` instead of `docless`. What do you think?